Validate operations (#177)
mirumee/ariadne-codegen: CHANGELOG.md

## UNRELEASED

- Added support for `Upload` scalar. Added support for file uploads to `AsyncBaseClient` and `BaseClient`.

> - Added validation of operations for compliance with provided schema.

Reviewer: ```suggestion
- Added validation of defined operations against the schema.
```
